scripts/ui-setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# clear out federation startup configs folder
+rm -rf $PWD/../setup
+
+# start bitcoind in the background
+bitcoind -regtest -daemon -fallbackfee=0.0004 -txindex -server -rpcuser=bitcoin -rpcpassword=bitcoin
+
+# start 5 guardians, kill processes on their ports if they are already running
+
+for ((ID = 0; ID < 5; ID++)); do
+  npx kill-port $((10000 + $ID))
+  cargo run --bin fedimintd $PWD/../setup/mint-$ID.json $PWD/../setup/mint-$ID.db $((10000 + $ID)) &
+done
+
+function kill_fedimint_processes {
+  pkill "fedimintd" || true
+}
+
+trap kill_fedimint_processes EXIT
```

Comment:
Might be nice if there was an argument here for how many servers you want. It could default to, say, 4?

Here is an example of another script with an argument wit a default like this https://github.com/fedimint/fedimint/blob/master/scripts/build.sh#L7